src/serac/physics/utilities/functional/tests/CMakeLists.txt

```diff
@@ -71,18 +51,8 @@ if(ENABLE_CUDA)
 
     set(functional_tests_cuda
         tensor_unit_tests_cuda.cu)
-        
-    foreach(filename ${functional_tests_cuda})
-        get_filename_component(test_name ${filename} NAME_WE)
-        
-        blt_add_executable(NAME        ${test_name}
-                           SOURCES     ${filename}
-                           OUTPUT_DIR  ${TEST_OUTPUT_DIRECTORY}
-                           DEPENDS_ON  gtest serac_functional ${functional_depends} cuda
-                           FOLDER      serac/tests)
-        blt_add_test(NAME          ${test_name}
-                     COMMAND       ${test_name}
-                     NUM_MPI_TASKS 1)
-    endforeach()
+
+    serac_add_tests( SOURCES ${functional_tests_cuda
```

Comment:
You missed a closing `}`

Reply:
That's what I get for rushing a PR before kid pick up time...